gsd-schema/osv-gsd/validation/schema.json

```diff
@@ -17,7 +17,8 @@
     "osv_gsd_schema_version": {
       "type": "string",
       "enum": [
-        "1.0.0"
+          "1.0.0",
+	  "1.0.1"
```

Comment:
This should be just 1.0.1, not all previous versions. (Each version requires the value to match the version used)

Reply:
I was wondering about that, since we're mostly backwards compatible should we force the latest, or be more relaxed?

Reply:
Oh, this is the "latest", not a specific version. My bad